pshtt/pshtt.py

```diff
@@ -287,6 +287,8 @@ def basic_check(endpoint):
 
             endpoint.redirect_immediately_to = immediate
             endpoint.redirect_immediately_to_www = re.match(r'^https?://www\.', immediate)
+            if endpoint.redirect_immediately_to_www is not None:
+                endpoint.redirect_immediately_to_www = True
```

Comment:
I agree, I think it would be better not to first store a `MatchObject`  in `redirect_immediately_to_www` and then change the value to a boolean. A slightly less terse alternative could be: 
```
www_match = re.match(r'^https?://www\.', immediate)
endpoint.redirect_immediately_to_www = www_match is not None
```